Source/WebCore/PAL/pal/spi/cocoa/NetworkSPI.h

```diff
@@ -58,6 +58,18 @@ nw_interface_t nw_path_copy_interface(nw_path_t);
 
 bool nw_settings_get_unified_http_enabled(void);
 
+void nw_parameters_set_server_mode(nw_parameters_t, bool);
+nw_parameters_t nw_parameters_create_webtransport_http(nw_parameters_configure_protocol_block_t, nw_parameters_configure_protocol_block_t, nw_parameters_configure_protocol_block_t, nw_parameters_configure_protocol_block_t);
+nw_protocol_options_t
+nw_webtransport_create_options(void);
+bool
```

Comment:
Let's put return values on the same line as the function so each function is one line.